setup.py

```diff
@@ -59,6 +59,9 @@ def process_ignore_tags(buffer):
             'sphinx>=3.0.0',
             'sphinx_rtd_theme>=0.4.3',
             'tox>=3.0.0'
+        ],
+        'usbmon': [
+            'enum34 ; python_version < "3.5"'
```

Comment:
Please don't include this and don't use Python 34 enums, but keep Scapy's dict for now. (Sure it would be nice to use Python 34+ enums everywhere... cf #3665)